partner_vat_unique/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Partner VAT Unique",
     "summary": "Module to make the VAT number unique for customers and suppliers.",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
```

Comment:
hi @AnizR !

nice pr here. just one comment : I think this kind of change should belong to a commit that says "Migration to 16.0"